doc/properties.xml

```diff
@@ -622,6 +622,32 @@ true]]></Example>
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IsMonogenicMonoid">
+  <ManSection>
+    <Prop Name = "IsMonogenicMonoid" Arg = "S"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+   <Description>
+     <C>IsMonogenicMonoid</C> returns <K>true</K> if the monoid
+     <A>S</A> is a monogenic monoid and it returns <K>false</K> if
+     it is not. <P/>
+
+     A monoid is <E>monogenic</E> if it is generated as a monoid
+     by a single element.  See also <Ref Prop = "IsMonogenicSemigroup"/>
+     and <Ref Attr = "IndexPeriodOfTransformation" BookName = "ref"/>.
```

Comment:
Please remove the reference to `IndexPeriodOfTransformation`, it might be hard to understand the relevance of this reference.